code/modules/mob/living/life.dm

```diff
@@ -24,6 +24,8 @@
 	if(stat != DEAD && !is_in_stasis())
 		handle_nutrition_and_hydration()
 		handle_immunity()
+	//Body temperature adjusts itself (self-regulation)
+	stabilize_body_temperature()
```

Comment:
This should be indented, otherwise dead mobs and mobs in stasis will still change their body temperature.